### PR TITLE
Feature food help

### PR DESCRIPTION
**Pull request by ArmantasPik**
-Added food page sorting
-Added paging
-Added food category filtering
-Fixed onGet paths

### CODE REVIEW

File: Pages/FoodCustomer/Food.cshtml.cs [hunk 1]
```diff
@@ -5,63 +5,135 @@
...
+            {
+                case "name_desc":
+                    foodIQ = foodIQ.OrderByDescending(s => s.FoodName);
+                    break;
+                case "Restaurant":
+                    foodIQ = foodIQ.OrderBy(s => s.ApplicationUser.RestaurantName);
+                    break;
+                case "resName_desc":
+                    foodIQ = foodIQ.OrderByDescending(s => s.ApplicationUser.RestaurantName);
+                    break;
```
Comment (medikornov) [inline Pages/FoodCustomer/Food.cshtml.cs:103]: looks little bit ugly:DDD

Comment (ArmantasPik) [reply Pages/FoodCustomer/Food.cshtml.cs:103]: :DD how to fix?

Comment (medikornov) [reply Pages/FoodCustomer/Food.cshtml.cs:103]: maybe needs a separate function

File: Pages/FoodPages/Index.cshtml.cs [hunk 2]
```diff
@@ -27,27 +28,101 @@ public IndexModel(FoodKept.Data.ShopContext context, UserManager<ApplicationUser
...
+                case "Discount":
+                    foodIQ = foodIQ.OrderBy(s => s.CurrentPrice.DiscountPercent);
+                    break;
+                case "discount_desc":
+                    foodIQ = foodIQ.OrderByDescending(s => s.CurrentPrice.DiscountPercent);
+                    break;
+                case "Quantity":
+                    foodIQ = foodIQ.OrderBy(s => s.Quantity);
+                    break;
+                case "quantity_desc":
```
Comment (medikornov) [inline Pages/FoodPages/Index.cshtml.cs:108]: same issue

**Review by medikornov — APPROVED** [review]

File: Pages/FoodPages/Index.cshtml.cs (hunk 2 above)
Comment (ArmantasPik) [reply Pages/FoodPages/Index.cshtml.cs:108]: I will put everything to a separate method.

**Review by 0rl1s — APPROVED** [review]
Nothing new to comment.

**Pull request merged**